### PR TITLE
PG-338: Calls count is not correct in PG-13.

### PR DESCRIPTION
**Pull request by ibrarahmad**
There is a duplicate query issue by this when the query plan is enabled.
Also fixes PG-331 - Defaults values for better presentation in PMM.

### CODE REVIEW

File: pg_stat_monitor.c [hunk 1]
```diff
@@ -1791,7 +1791,12 @@ pg_stat_monitor_internal(FunctionCallInfo fcinfo,
 			tmp = e->counters;
 			SpinLockRelease(&e->mutex);
 		}
-		if (!IsBucketValid(bucketid))
+
+        /* In case plan is enable, there is no need to show 0 planid query */
```
Comment (LenzGr) [inline pg_stat_monitor.c:1795]: Minor nit:
```suggestion
        /* In case that query plan is enabled, there is no need to show 0 planid query */
```

**Comment by EngineeredVirus** [comment]
LGTM

**Comment by EngineeredVirus** [comment]
LGTM

**Review by EngineeredVirus — COMMENTED** [review]
LGTM

**Pull request merged**